adam-core/src/main/scala/org/bdgenomics/adam/rdd/GenomicRDD.scala

```diff
@@ -244,7 +372,7 @@ trait GenomicRDD[T, U <: GenomicRDD[T, U]] {
 
       // repartition yonder our data
       // TODO: this should repartition and sort within the partition
-      binKeyedRdd.repartitionAndSortWithinPartitions(ManualRegionPartitioner(bins.numBins))
+      binKeyedRdd.repartitionAndSortWithinPartitions(new ReferenceRegionRangePartitioner(bins.numBins))
```

Comment:
This was actually @fnothaft's code, but I merged two partitioners that were doing the exact same thing. I don't know if we can remove the `TODO`.